src/components/common/ui/form/text-area-mui/TextArea.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  FormControl,
+  FormHelperText,
+  InputLabel,
+  OutlinedInput,
+} from '@mui/material';
+import { SxProps, Theme } from '@mui/material/styles';
+import { useField } from 'formik';
+
+import mergeSx from '@/lib/utils/MergeSxStylesUtil';
+
+import * as styles from './TextArea.styles';
+
+interface TextAreaProps {
+  name: string;
+  placeholder?: string;
+  label?: string;
+  size?: 'medium' | 'small';
+  disabled?: boolean;
+  showRemark?: boolean;
+  sx?: SxProps<Theme>;
+}
+
+const TextArea: React.FC<TextAreaProps> = ({
+  name,
+  placeholder,
+  label,
+  size = 'medium',
+  disabled = false,
+  showRemark = false,
+  sx,
+}) => {
+  const [field, { touched, error }] = useField(name);
+
+  const state = touched && error ? 'error' : 'default';
+
```

Comment:
i`m not sure about this one, but maybe create boolean variable 'isStateError', so you can rewrite code as: 

`const isStateError =  touched && error;`
`const state =isStateError ? 'error' : 'default';`


so, you can also use it in next part of code:
 `{showRemark && (`
       ` <FormHelperText sx={styles.errorRemark}>`
         ` {touched && error}`
      `  </FormHelperText>`
   `  )}`
as         
      `{showRemark && (`
       ` <FormHelperText sx={styles.errorRemark}>`
          `{isStateError}`
        `</FormHelperText>`
      `)}`